ng-sample/app/examples/renderer-test.ts

```diff
@@ -59,7 +59,9 @@ export class RendererTest {
             'segmentedBarItems': [
                 {'title': 'first'},
                 {'title': 'second'},
-                {'title': 'third'}
+                {'title': 'third'},
+                {'title': 'fourth'},
+                {'title': 'fifth'}
```

Comment:
Adding these makes the `RendererTest` work 100%. Previously using the `ListPicker` and setting the value to 4 or 5 would crash the example due to selectedIndex out of range.